tests/migration/online_migration/zypper_migration.pm

```diff
@@ -66,9 +66,9 @@ sub run {
     while ($out) {
         diag "out=$out";
         if ($out =~ $zypper_migration_target) {
-            my $version = get_var("VERSION");
-            $version =~ s/-/ /;
-            if ($out =~ /(\d+)\s+\|\s?SUSE Linux Enterprise.*?$version/m) {
+            my $target_version = get_var("TARGET_VERSION", get_required_var("VERSION"));
+            $target_version =~ s/-/ /;
```

Comment:
Is this really needed?

Reply:
Reason: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14138

Reply:
I mean the `s/-/ /;` as version nor target_version does not contain any `-` character.

Reply:
It's a good point, no worries. And It is somehow related to slem, but it was added some time ago :)